karma.conf.js

```diff
@@ -2,13 +2,7 @@
 const os = require("os");
 const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
 
-if (os.platform === "linux" && os.arch() === "arm64") {
-
-    // For arm64 architecture, install chromium-browser using "apt-get install chromium-browser"
```

Comment:
We lost the comment here while autofixing [unicorn/prefer-ternary](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-ternary.md).

Reply:
Good catch; Issue has been reported at https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1757

Reply:
Fixed